lib/database.js

```diff
@@ -173,6 +173,20 @@ class Database {
     return column;
   }
 
+  async count(table, ...conditions) {
+    const { clause, args } = buildWhere(conditions);
+    const sql = [
+      `SELECT count(*) FROM "${table}"`,
+      clause ? `WHERE ${clause}` : '',
+    ];
+
+    const {
+      rows: [{ count }],
+    } = await this.query(sql.join(' '), args);
+
+    return parseInt(count);
```

Comment:
@tshemsedinov 
this aren't working. same issue as I mention in https://github.com/metarhia/metasql/issues/201
since select wraps each field into double quotes it is treated as a column
`column "count(*)" does not exist`

Reply:
@tshemsedinov committed a little tweak to select method that resolves this issue